core/bourbon/helpers/_contrasting-color.scss

```diff
@@ -0,0 +1,7 @@
+@function contrasting-color($background-color, $light-color: #000, $dark-color: #fff) {
```

Comment:
Arg, this is very nitpicky but I feel like `background-color` is a misnomer. what if this is the `text-color` or a `shadow-color`